x/tokenomics/keeper/keeper_settle_pending_claims_test.go

```diff
@@ -241,6 +241,61 @@ func (s *TestSuite) TestSettlePendingClaims_ClaimSettled_ProofRequiredAndProvide
 	require.Equal(t, s.expectedComputeUnits, expectedEvent.ComputeUnits)
 }
 
+func (s *TestSuite) TestClaimSettlement_ClaimSettled_ProofRequiredAndProvided_ViaProbability() {
+	// Retrieve default values
+	t := s.T()
+	ctx := s.ctx
+	sdkCtx := cosmostypes.UnwrapSDKContext(ctx)
+	sharedParams := s.keepers.SharedKeeper.GetParams(ctx)
+
+	// Set the proof parameters such that s.claim requires a proof because the
+	// proof_request_probability is 100%. This is accomplished by setting the
+	// proof_requirement_threshold to exceed s.expectedComputeUnitss, which
```

Comment:
```suggestion
	// proof_requirement_threshold to exceed s.expectedComputeUnits, which
```

Reply:
Thanks for catching this. The "s" key switch on my keyboard was going out but I've since replaced it. :mechanical_arm: